### ISSUE TITLE
.org redesign: community [content/copy]

### ISSUE THREAD

**Issue by marisamorby** [gatsbyjs/gatsby#9726]
## Goal

As we're going through the .org redesign, we'll be updating the copy on each of the pages. We want to make sure that the right copy is listed for our community, and that the copy also fulfills our marketing goals.

The sections for the homepage will be: 
- Code of Conduct
- How to Contribute
- other community specific information or guides we want to surface (this is an ongoing conversation)

## Next steps
- @shannonbux: are there specific questions you've heard about how to contribute or other community specific issues that we could address with specific copy on a separate community page
- @lindaleebumblebee: is there specific marketing copy we should highlight on this page that you'd like to let the community know about?

**Comment by lindaleebumblebee** [comment]
It'd also be great to include a link to a page that has 'How to connect' or something to that effect that then lists all the Gatsby Days and meetups that are planned. We need to build that page as well, but we should make it easy for the community to connect with us and each other.

**Comment by marisamorby** [comment]
That's a great idea @lindaleebumblebee. I'll add it to the Epic so we can log it!

**Comment by shannonbux** [comment]
You know, I think @jlengstorf and the rest of the developers who have been
on pair programming sessions would have the best summary of contribution
- FAQs
- other pain points or problems we could solve

I don't really have any research on this; almost no interviewees have
commented on the contribution process other than to say this:
- Kyle is super positive and quickly responds
- I wish all the stuff buried in issues was easier to find in the docs
(which I've been working on)
- Why don't we have a stack overflow or Jira? It's easier to manage
questions there
- Jason and co. have been awesome to work with on pair programming
- The swag after first PR and becoming a "maintainer" makes new people feel
really special and feel super responsible to keep coming back

So that's what I know!

The rest I know is pain points from my end:
- Almost no one reads the style guide or docs decision tree before creating
docs related PRs and issues even if I link to them in the issue and say you
must do this or it will not be merged. So then I just have to remind
everyone to really do it. And sometimes they still don't, so it makes me
wonder why. I don't know why! Could be that they don't have time, it's
boring, it's hard to follow, grammar isn't everyone's love.

**Comment by marisamorby** [comment]
@shannonbux: this is a good callout. We'll be creating the copy on the community page from scratch, so getting more info from the pair programming sessions would be great!
